pkgs/by-name/pa/panda3ds/package.nix

```diff
@@ -0,0 +1,199 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+
+  xorg,
+  kdePackages,
+  cmake,
+  pkg-config,
+  imagemagick,
+
+  # (vulkan is currently not supported with qtgui and will cause a segfault, so don't even bother trying to compile it)
+  withVulkan ? stdenv.hostPlatform.isLinux && !withQtGui,
+  withOpenGL ? stdenv.hostPlatform.isLinux || withQtGui,
+  withMetal ? stdenv.hostPlatform.isDarwin,
+  withLuaJIT ? true,
+  withQtGui ? false,
+  withQtWayland ? false, # (broken)
+  withDiscordRPC ? true,
+  withRenderdoc ? false,
+  sse4Support ? stdenv.hostPlatform.sse4_1Support,
+
+  SDL2,
+  libGL,
+  vulkan-headers,
+  vulkan-loader,
+  glslang,
+  wayland,
+  # misc. rt dep:
+  pulseaudio,
+
+  # msc. de-vendored deps
+  boost,
+  discord-rpc,
+  rapidjson, # required transitive: discord-rpc dependency
```

Comment:
Is this not propagated automatically? That's annoying.

Reply:
(should be, i added it for the vendored discord-rpc before i removed it)